### DISCUSSION TITLE
raw content viewer

### DISCUSSION THREAD

**Discussion by PiotrekUnityDeveloper** [community/community#63315] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Feature Area

Issues

### Body

my program uses csharp to download a string from a raw repo link. Precisely, im using WebClient (used in .net) to read content of the raw page, it looks like the link crashed after a few of those requests and the raw button doesnt even work from the browser. i tried logging in again, didnt help it.

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by PiotrekUnityDeveloper** [comment]
https://raw.githubusercontent.com/PiotrekUnityDeveloper/totalkiller/main/versions

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @PiotrekUnityDeveloper ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Projects and Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!
